prow/plugins.yaml

```diff
@@ -11,6 +11,7 @@ triggers:
   - google/kubeflow
   - tensorflow/k8s
   - GoogleCloudPlatform/k8s-multicluster-ingress
+  - containerd/cri-containerd
```

Comment:
I think you'll need to re-configure webhooks, are you a repo admin?

Reply:
Yes this will need webhook configuration

Reply:
Can we merge this first? I'll deal with the webhook.